### PR TITLE
feat(rest-api-client): add `space.deleteGuests()` method

### PR DESCRIPTION
**Pull request by tuanphamcybozu**
## Why
- Support `deleteGuests` method to be able to delete guest users from Kintone.

## What
- Add new method `deleteGuests`
- Add unit test
- Add demo script

## How to test
- Check test spec

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/js-sdk/blob/master/CONTRIBUTING.md)
- [x] Updated documentation if it is required.
- [X] Added tests if it is required.
- [x] Passed `pnpm lint` and `pnpm test` on the root directory.

### CODE REVIEW

File: packages/rest-api-client/docs/space.md [hunk 1]
```diff
@@ -189,3 +190,21 @@ An empty object.
 #### Reference
 
 - https://kintone.dev/en/docs/kintone/rest-api/spaces/update-thread/
+
+### deleteGuests
+
+Deletes a Guest user from Kintone. If you would like to remove a user from a Guest Space without deleting their account, use the Update Guest Members API.
```
Comment (hung-cybo) [inline packages/rest-api-client/docs/space.md:196]: Notes: we need to add the link to [Update Guest Members API] after we finish that PBI.

**Review by hung-cybo — APPROVED** [review]
LGTM

**Review by tasshi-me — APPROVED** [review]
LGTM!

**Review by tasshi-me — APPROVED** [review]
LGTM!

**Pull request merged**